inginious/agent/docker_agent/__init__.py

```diff
@@ -364,9 +373,14 @@ def __new_job_sync(self, message: BackendNewJob, future_results):
 
         # Run the container
         try:
+            # get process *NOFILE* resource limits
+            p = psutil.Process()
```

Comment:
psutil docs indicates it behaves the same as standard library ``resources.getrlimit`` if we don't care about the PID we want to apply the limit to. I guess that there is no need to rely on psutil for this ?